Source/JSONParser.swift

```diff
@@ -131,7 +131,7 @@ extension GenericJSONParser {
         return Character(UnicodeScalar(currentChar))
     }
 
-    private func parseSymbol(_ target: StaticString, @autoclosure _ iftrue: Void -> JSON) throws -> JSON {
+    private func parseSymbol(_ target: StaticString, _ iftrue: @autoclosure Void -> JSON) throws -> JSON {
```

Comment:
Hound only analyzes diffs. Perhaps we should break this into multiple lines to appease it?

I'm a little ambivalent about the whole line length requirement tbh.